src/reviewer.py

```diff
@@ -0,0 +1,33 @@
+import os
+from github_utils import get_pull_request_diff, post_review_comment
+from llm_utils import analyze_code_changes
+
+class AICodeReviewer:
+    def __init__(self):
+        self.openai_api_key = os.getenv('GEMINI_API_KEY')
+        self.github_token = os.getenv('GIT_TOKEN')
+
+    def review_pull_request(self):
+        try:
```

Comment:
The `review_pull_request` method lacks error handling for specific exceptions within the try block. It's better to catch specific exceptions and handle them appropriately instead of a general `Exception`.

```suggestion
Use specific exception handling for `get_pull_request_diff` and `analyze_code_changes`.
```